testcaseUI/testcase-backend/backend/model.py

```diff
@@ -6,6 +6,7 @@
 from git import Repo
 import os
 import git
+from pathlib import Path
```

Comment:
Unused import?

Reply:
The import was missing before. Therefore I added it :)